cpp/examples/parquet_io/common_utils.cpp

```diff
@@ -0,0 +1,175 @@
+
+/*
+ * Copyright (c) 2024, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "common_utils.hpp"
+
+#include <cudf/concatenate.hpp>
+#include <cudf/io/types.hpp>
+#include <cudf/join.hpp>
+#include <cudf/table/table_view.hpp>
+
+#include <rmm/mr/device/device_memory_resource.hpp>
+#include <rmm/mr/device/owning_wrapper.hpp>
+#include <rmm/mr/device/pool_memory_resource.hpp>
+
+#include <fmt/color.h>
```

Comment:
What library is this from? I'd like to minimize library dependencies in the examples if possible.

Reply:
`fmtlib`. I think it's already a dependency of [libcudf](https://github.com/rapidsai/cudf/pull/14355) so the example doesn't add any additional dependencies

Reply:
But we want to remove fmt & spdlog. Let’s avoid this dependency.

https://github.com/rapidsai/cudf/issues/16951

https://github.com/rapidsai/build-planning/issues/104

Reply:
Sounds good.